diff -Nuard pan-0.14.2.orig/configure.in pan-0.14.2/configure.in
--- pan-0.14.2.orig/configure.in	2003-08-31 18:27:33.000000000 +0200
+++ pan-0.14.2/configure.in	2004-02-22 11:32:11.358037664 +0100
@@ -7,6 +7,7 @@
 AC_INIT([pan],[0.14.2],[http://bugzilla.gnome.org/enter_bug.cgi?product=Pan])
 AC_DEFINE(VERSION_TITLE,["This is not a psychotic episode. It's a cleansing moment of clarity."],[Release Title])
 AC_PREREQ(2.50)
+AC_CONFIG_AUX_DIR([.])
 AC_CONFIG_SRCDIR(pan/pan.c)
 AC_PROG_INTLTOOL([0.21])
 
