--- pan-0.10.0/configure.in.orig	Fri Aug 17 04:06:58 2001
+++ pan-0.10.0/configure.in	Fri Aug 17 21:12:44 2001
@@ -35,7 +35,7 @@
 
 dnl i18n
 ALL_LINGUAS="ca da de en_GB es fi fr ga gl hu it ja ko lt no nl pl pt_BR ru sk sl sp sr sv tr uk"
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT
 
 
 dnl ***
@@ -187,6 +187,7 @@
            pan.spec
            pan-images.spec
            macros/Makefile
+           intl/Makefile
            po/Makefile.in
            pan/Makefile
            pan/base/Makefile
--- pan-0.10.0/Makefile.am~	Wed Aug  1 00:57:49 2001
+++ pan-0.10.0/Makefile.am	Fri Aug 17 21:51:55 2001
@@ -1,4 +1,4 @@
-SUBDIRS = macros po gmime pan docs tests
+SUBDIRS = intl macros po gmime pan docs tests
 
 EXTRA_DIST = ANNOUNCE CREDITS ChangeLog pan.desktop pan.spec pan-images.spec pan.png
 
