diff -Nur pan-0.14.0.orig/configure.in pan-0.14.0.chng/configure.in
--- pan-0.14.0.orig/configure.in	Mon May  5 20:23:57 2003
+++ pan-0.14.0.chng/configure.in	Tue Jun 24 14:09:16 2003
@@ -68,7 +68,7 @@
 AC_SUBST(XML_LIBS)
 
 dnl Need Gnet
-AM_PATH_GNET(1.1.5,
+AM_PATH_GNET_2_0(1.1.5,
   [LIBS="$LIBS $GNET_LIBS" CFLAGS="$CFLAGS $GNET_CFLAGS"],
   AC_MSG_ERROR([Cannot find GNet: Is gnet-config in path?]))
 
diff -Nur pan-0.14.0.orig/pan/dialogs/dialog-newuser.c pan-0.14.0.chng/pan/dialogs/dialog-newuser.c
--- pan-0.14.0.orig/pan/dialogs/dialog-newuser.c	Thu Mar 27 14:58:14 2003
+++ pan-0.14.0.chng/pan/dialogs/dialog-newuser.c	Tue Jun 24 14:36:18 2003
@@ -28,7 +28,7 @@
 
 #include <ctype.h>
 #include <unistd.h>
-#include <gnet/gnet.h>
+#include <gnet-2.0/gnet.h>
 
 #include <pan/base/debug.h>
 #include <pan/base/pan-config.h>
@@ -97,7 +97,7 @@
 	char * retval = NULL;
 	debug_enter ("get_fqdn");
 
-	retval = gnet_inetaddr_gethostname ();
+	retval = gnet_inetaddr_get_host_name ();
 
 	debug_exit ("get_fqdn");
 	return retval;
diff -Nur pan-0.14.0.orig/pan/smtp.c pan-0.14.0.chng/pan/smtp.c
--- pan-0.14.0.orig/pan/smtp.c	Wed Apr 23 20:29:28 2003
+++ pan-0.14.0.chng/pan/smtp.c	Tue Jun 24 14:36:11 2003
@@ -24,7 +24,7 @@
 #include <unistd.h>
 
 #include <glib.h>
-#include <gnet/gnet.h>
+#include <gnet-2.0/gnet.h>
 
 #include <gmime/gmime-utils.h>
 
@@ -279,7 +279,7 @@
 smtp_handshake (StatusItem   * item,
                 PanSocket    * sock)
 {
-	char * my_name = gnet_inetaddr_gethostname ();
+	char * my_name = gnet_inetaddr_get_host_name ();
 
 	status_item_emit_status (item, _("Handshaking with mail server"));
 
diff -Nur pan-0.14.0.90/pan/sockets.c pan-0.14.0.90-patch/pan/sockets.c
--- pan-0.14.0.90/pan/sockets.c	2003-07-02 07:02:18.000000000 +0200
+++ pan-0.14.0.90-patch/pan/sockets.c	2003-07-12 14:47:16.000000000 +0200
@@ -30,7 +30,7 @@
 #include <time.h> /* for time_t */
 
 #include <glib.h>
-#include <gnet/gnet.h>
+#include <gnet-2.0/gnet.h>
 
 #include <pan/base/debug.h>
 #include <pan/base/log.h>
@@ -154,7 +154,7 @@
 	}
 	else
 	{
-		sock->gio_channel = gnet_tcp_socket_get_iochannel (sock->gnet_socket);
+		sock->gio_channel = gnet_tcp_socket_get_io_channel (sock->gnet_socket);
 		if (g_io_channel_get_encoding (sock->gio_channel) != NULL)
 			g_io_channel_set_encoding (sock->gio_channel, NULL, NULL);
 		g_io_channel_set_flags (sock->gio_channel, G_IO_FLAG_NONBLOCK, NULL);
